plugin/src/main/groovy/com/novoda/staticanalysis/internal/detekt/DetektConfigurator.groovy

```diff
@@ -3,12 +3,16 @@ package com.novoda.staticanalysis.internal.detekt
 import com.novoda.staticanalysis.StaticAnalysisExtension
 import com.novoda.staticanalysis.internal.Configurator
 import com.novoda.staticanalysis.internal.Violations
+import org.gradle.api.GradleException
 import org.gradle.api.NamedDomainObjectContainer
 import org.gradle.api.Project
 import org.gradle.api.Task
 
 class DetektConfigurator implements Configurator {
 
+    private static final String DETEKT_PLUGIN = 'io.gitlab.arturbosch.detekt'
+    private static final String DETEKT_NOT_APPLIED = 'The detekt plugin is configured but not applied. Please apply the plugin in your build script.'
```

Comment:
Maybe some more informations would help users. At least a "refer to the Detekt documentation at https://github.com/arturbosch/detekt for further details"